calendars/scipy.yaml

```diff
@@ -27,3 +27,20 @@ events:
         # seconds, minutes, hours, days, weeks, months, years
         days: 28
       until: 2025-01-01 00:00:00 +00:00
+
+  - summary: Monthly SciPy New Contributor Meeting
+    description: |
+      The monthly call for new contributors to SciPy 🎉
+
+      All are welcome.
+
+      Meeting notes: https://hackmd.io/@melissawm/H15ahr5wq
```

Comment:
At some point maybe we should have better urls. What do you think? Very low prio so keeping this for later.

Reply:
We could have a hackmd workspace like matplotlib has one. However I just learned yesterday that hackmd is not accessible - so maybe we can revisit this later?

Reply:
Yes OC. I approved, but don't have permissions to merge. Otherwise it would be in already 😉

We can use the workspace we have for scientific-python I suppose.